bzzeth/bzzeth.go

```diff
@@ -418,11 +417,11 @@ DELIVERY:
 	if err == nil && deliveredCnt < total {
 		err := p.Send(ctx, &BlockHeaders{Rid: uint32(msg.Rid)})
 		if err != nil {
-			p.logger.Error("could not send empty BlockHeader", "err", err)
+			return fmt.Errorf("could not send empty BlockHeader: %w", err)
```

Comment:
i am assuming that here `peer` is the ethereum node that expects the block headers. the send fails here, don't we want to break too?